docker/Dockerfile.gpu

```diff
@@ -2,10 +2,10 @@ FROM nvidia/cuda:9.0-cudnn7-devel
 
 COPY install.sh /
 
-RUN /install.sh
+RUN /install.sh && pip install mxnet-cu90 && mkdir /mxnet_model_server
```

Comment:
Why not mxnet-cu91mkl?

Reply:
This needs a little more work to change. Created an issue #335  for tracking this.